net-mail/gnubiff/gnubiff-2.2.17-r3.ebuild

```
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools
+
+DESCRIPTION="A mail notification program"
+HOMEPAGE="http://gnubiff.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3+-with-openssl-exception"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="debug fam nls password"
+
+RDEPEND="
+	>=x11-libs/gtk+-3:3
+	>=gnome-base/libglade-2.3
+	dev-libs/popt
+	password? ( dev-libs/openssl:0= )
+	fam? ( virtual/fam )
+	x11-libs/libX11
+	x11-libs/pango
+	x11-libs/gdk-pixbuf
+"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+"
+
+DOCS="AUTHORS ChangeLog NEWS README THANKS TODO"
+
+src_prepare() {
+	eapply "${FILESDIR}/${PN}-2.2.17-clang.patch"
+	eapply_user
+	eautoreconf
+}
+
+src_configure() {
+	# note: --disable-gnome is to avoid deprecated gnome-panel-2.x
+	econf \
+		--disable-gnome \
+		$(use_enable debug) \
+		$(use_enable nls) \
+		$(use_enable fam) \
+		$(use_with password) \
+		$(use_with password password-string ${RANDOM}${RANDOM}${RANDOM}${RANDOM})
```

```suggestion
	local myeconfargs=(
		--disable-gnome
		$(use_enable debug)
		$(use_enable nls)
		$(use_enable fam)
		$(use_with password)
		$(use_with password password-string ${RANDOM}${RANDOM}${RANDOM}${RANDOM})
	)
	econf "${myeconfargs[@]}"
```

Also, is that something save to set the password-string with all these RANDOM numbers? Just asking.

---

I have no idea what that's for, this isn't my code, I'm just trying to give it an extra lease on life.